libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/DataStore.java

```diff
@@ -128,6 +128,29 @@ public List<T> getItemsWithAttributeLessThanOrEqualValue(
                 .toList();
     }
 
+    public List<T> getItemsByRange(
```

Comment:
Is this right? I thought the idea was that we are using the bash script to run the scan to get the list of VCs, and then we don't need to rescan in the lambda? (i.e. we just pass the lambda a list of keys to resend)